build_debian.sh

```diff
@@ -280,7 +280,8 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     ipmitool                \
     ndisc6                  \
     makedumpfile            \
-    conntrack
+    conntrack               \
+    python3-pip
```

Comment:
can you move python-pip to here together for better maintenance later?

Reply:
I noticed, we install  python-pip explicitly in many sonic dockers including slave docker. But I do not see installing python-pip explicitly in sonic base image. Note, Pyhton2.7 comes with pip inbuild. I think, as of now it is better to add python-pip in build_debian.sh .